e2etest/unbondingwatcher_e2e_test.go

```diff
@@ -170,3 +199,117 @@ func TestActivatingDelegation(t *testing.T) {
 		return resp.BtcDelegation.Active
 	}, eventuallyWaitTimeOut, eventuallyPollTime)
 }
+
+// TestActivatingAndUnbondingDelegation tests that delegation will eventually become UNBONDED given that
+// both staking and unbonding tx are in the same block.
+// In this test, we include both staking tx and unbonding tx in the same block.
+// The delegation goes through "VERIFIED" → "ACTIVE" → "UNBONDED" status throughout this test.
+func TestActivatingAndUnbondingDelegation(t *testing.T) {
+	//t.Parallel()
```

Comment:
for some reason, there might be an unwanted dependancy between e2e test, making it sequential until, will investigate (related to btc frequent headers maybe)

Reply:
"works on my machine" type of thing :sweat_smile: